src/VscPreviewExtension/.gitignore

```diff
@@ -0,0 +1,6 @@
+node_modules/
+out/
+src/*
```

Comment:
I wanted to add the folder but not the files , but it didn't work

---
In reply to: [74028182](https://github.com/dotnet/docfx/pull/589#discussion_r74028182) [](ancestors = 74028182)